test/smoke_test.rb

```diff
@@ -23,6 +23,8 @@ class MovieSearch < FortyFacets::FacetSearch
   facet [:studio, :country], name: 'Country'
   facet [:studio, :status], name: 'Studio status'
   facet [:studio, :producers], name: 'Producers'
+  sql_facet({ classic: 'year <= 1980', non_classic: 'year > 1980' }, { name: 'Classic', path: :classic })
```

Comment:
Line is too long. [105/80]<br>Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.